core/storage/user/gae_models_test.py

```diff
@@ -2245,6 +2245,94 @@ def test_get_translation_reviewer_user_ids(self):
         self.assertTrue(self.USER_ID_1 in translation_reviewer_ids)
         self.assertTrue(self.USER_ID_2 in translation_reviewer_ids)
 
+    def test_get_num_reviewers_for_translations_in_lang_code_if_no_reviewers(
+            self):
+        number_of_translation_reviewers = (
+            user_models.UserContributionRightsModel
+            .get_number_of_reviewers_for_translations_in_lang_code('hi')
+        )
+
+        self.assertEqual(number_of_translation_reviewers, 0)
+
+    def test_get_num_of_reviewers_for_translations_in_lang_code_if_same_lang(
+            self):
+        user_models.UserContributionRightsModel(
+            id=self.USER_ID_1,
+            can_review_translation_for_language_codes=['hi'],
+            can_review_voiceover_for_language_codes=[],
+            can_review_questions=False).put()
+        user_models.UserContributionRightsModel(
+            id=self.USER_ID_2,
+            can_review_translation_for_language_codes=['hi'],
+            can_review_voiceover_for_language_codes=[],
+            can_review_questions=True).put()
+
+        number_of_translation_reviewers = (
+            user_models.UserContributionRightsModel
+            .get_number_of_reviewers_for_translations_in_lang_code('hi')
+        )
+
+        self.assertEqual(number_of_translation_reviewers, 2)
+
+    def test_get_num_of_reviewers_for_translations_in_lang_code_if_diff_lang(
+            self):
+        user_models.UserContributionRightsModel(
+            id=self.USER_ID_1,
+            can_review_translation_for_language_codes=['en'],
+            can_review_voiceover_for_language_codes=[],
+            can_review_questions=False).put()
+        user_models.UserContributionRightsModel(
+            id=self.USER_ID_2,
+            can_review_translation_for_language_codes=['en'],
+            can_review_voiceover_for_language_codes=[],
+            can_review_questions=True).put()
+
+        number_of_translation_reviewers = (
+            user_models.UserContributionRightsModel
+            .get_number_of_reviewers_for_translations_in_lang_code('hi')
+        )
+
+        self.assertEqual(number_of_translation_reviewers, 0)
+
+    def test_get_num_reviewers_for_translations_in_lang_code_if_invalid_lang(
```

Comment:
Maybe actually add a reviewer with a valid lang to compare, otherwise this sort of devolves to the 'empty' test case.